apistar_prometheus/components.py

```diff
@@ -30,7 +30,7 @@ def track_request_start(self, method, handler=None):
 
         REQUESTS_INPROGRESS.labels(method, handler_name).inc()
 
-    def track_request_end(self, method, handler, response):
+    def track_request_end(self, method, handler, status_code):
```

Comment:
Technically, this is a breaking change, but I suppose no one should be using `Prometheus` directly and this package is still pre-1.0 so that's OK.